iosHackthonApp/Modules/Features/AddMeal/Sources/AddMeal/AddMealViewModel.swift

```diff
@@ -9,30 +9,31 @@
 import shared
 import Components
 import Location
+import Extensions
 
 protocol AddMealViewModelProtocol {
```

Comment:
What are we using these protocols for? Can't see it being used anywhere other than this file?

Reply:
Yeah, we aren't.
Think I wrote that once upon a time with the intention that I might use it for writing tests...and then didn't

Reply:
No problem! Just wondering if I missed something